xml/http_servers.xml

```diff
@@ -3283,4 +3283,12 @@
     <param pos="0" name="service.product" value="Symantec Endpoint Protection Manager"/>
     <param pos="0" name="service.family" value="Symantec Endpoint Protection Manager"/>
   </fingerprint>
+  <fingerprint pattern="^Intel\(R\) Active Management Technology\s(\d+\.\d+\.\d+\.\d+|\d+\.\d+\.\d+|\d+\.\d+)">
+    <description>Intel(R) Active Management Technology</description>
+    <example>Intel(R) Active Management Technology 7.1.86</example>
```

Comment:
Any extractions should have tests, IMO.  Will add this as part of my PR against yours.